data-prepper-plugins/date-processor/src/main/java/com/amazon/dataprepper/plugins/processor/date/DateProcessor.java

```diff
@@ -29,18 +29,26 @@
 @DataPrepperPlugin(name = "date", pluginType = Processor.class, pluginConfigurationType = DateProcessorConfig.class)
 public class DateProcessor extends AbstractProcessor<Record<Event>, Record<Event>> {
     private static final Logger LOG = LoggerFactory.getLogger(DateProcessor.class);
-    private static final ZoneId OUTPUT_TIMEZONE = ZoneId.systemDefault();
-    private static final String OUTPUT_FORMAT = "yyyy-MM-dd'T'HH:mm:ss.SSSXXX";
+    static final String OUTPUT_FORMAT = "yyyy-MM-dd'T'HH:mm:ss.SSSXXX";
```

Comment:
Let's restrict always try to keep the scope as reduced as possible. We can mark this as private.